aptos-indexer-processors-sdk/sdk/src/common_steps/version_tracker_step.rs

```diff
@@ -0,0 +1,164 @@
+use crate::{
+    traits::{
+        pollable_async_step::PollableAsyncRunType, NamedStep, PollableAsyncStep, Processable,
+    },
+    types::transaction_context::TransactionContext,
+    utils::errors::ProcessorError,
+};
+use ahash::AHashMap;
+use anyhow::Result;
+use async_trait::async_trait;
+use tracing::info;
+
+const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
+
+#[async_trait]
+pub trait ProcessorStatusSaver {
```

Comment:
just one implementation (the one in [this PR](https://github.com/aptos-labs/aptos-indexer-processor-example/pull/6)) otherwise we'd need to have an entirely other processor and implementation for the BF

Reply:
How would the trait know if it's a backfill processor? One idea is we could have 2 implementations, then in the processor chaining, if `backfilll_alias` is present, use the backfill implementation, otherwise use the processor status implementation.

Reply:
The two impl approaches is a great idea!